eradiate/scenes/measure.py

```diff
@@ -388,24 +389,21 @@ class PerspectiveCameraMeasure(Measure):
 
         Unit-enabled field (default: cdu[length]).
 
-    ``zenith`` (float):
-        Zenith angle. Default value: 0 deg.
-
-        Unit-enabled field (default: cdu[angle]).
-
-    ``azimuth`` (float):
-        Azimuth angle value. Default value: 0 deg.
+    ``origin`` (array[float]):
+        A 3-element vector specifying the position of the camera.
+        Default: [0, 0, 0] m.
 
-        Unit-enabled field (default: cdu[angle]).
+        Unit-enabled field (default: cdu[length]).
 
-    ``distance`` (float):
-        Distance from the ``target`` point to the camera.
-        Default: 1 km.
+    ``up`` (array[float]):
+        A 3-element vector specifying the vertical direction of the camera.
+        Default: [0, 0, 0].
```

Comment:
I don't think this default can work (at least I don't see how). A better value would be [0, 0, 1] IMO. Also I'm a bit uncomfortable with calling this "vertical" because, well... If you set it to something else than [0, 0, 1], it's not the vertical anymore (assuming gravity is oriented to -Z, ofc).

Sticking with "up" is better IMO. Something like: "A 3-element vector specifying the "up" direction of the camera."